essential_reading/DataSources/Data_S3.md

```diff
@@ -0,0 +1,223 @@
+Execute full data read operations only if this notebook is being hosted and run out of the same AWS center where the data lives.
+
+:::
+
+:::{sidebar}
+
+The good news about some of the larger science-oriented libraries (xarray, dask, pandas, zarr, etc), is that
+they can automatically handle the `fsspec` operations for you **IF YOUR ACCESS IS ANONYMOUS**.
+This is a convenience, but is a special case for read-only data where `anon=True` and `requestor_pays=False`.
+
+Note that this is a feature of
+[specific libraries](https://filesystem-spec.readthedocs.io/en/latest/#who-uses-fsspec),
+and doesn't work everywhere.
+
+Because it isn't universally available, and only applies to anonymous reads, examples in HyTEST will
+always explicitly plumb `fsspec` 'longhand' using `get_mapper()`. You may see example code elsewhere
+that takes the shortcut if it is available.
+
+:::
+
+
+## Credentialed Access
+
+For most data storage within the HyTEST workflows, access will not be anonymous.
+Permissions are set by the owners of that data, using credentials assigned to an AWS 'profile'.
+
+Profile credentials are usually stored outside of the Python program (typically in a master file in
+your ``HOME` folder on the compute/jupyter server).
+You need to have this set up beforehand, and is usually achieved by copying specific credential files into the right spot.
+A common mechanism to handle the profile configuration is with the `aws`
+[command line interface](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/configure/index.html).
+
+To create a new AWS profile:
+
+```sh
+> aws configure --profile osn-renci
+AWS Access Key ID :
+AWS Secret Access Key :
+Default region name : us-east-1
+Default output format: json
+```
+
+The first two prompts will ask for key/security information you were assigned for access to object storage.
+Answer last questions with "us-east-1" (region) and "json" (format).
+
+Note that this configuration is specific to the OSN 'pod' storage.
+Your profile name and region may be different.
+
+## Endpoints
+
+For storage operations, the AWS system needs the web address of the access point, or _endpoint_
+where it should address filesystem operations. If your storage is completely within the Amazon
+ecosystem, you will likely not need to specify an endpoint.
+However, for 3rd-party storage (such as the OSN pod), you will need to explicitly declare the
+endpoint when the filesystem is first referenced using `fsspec`.
+
+```python
+fs_write = fsspec.filesystem(
+    's3',
+    profile='osn-renci',  ## This is the profile name you configured above.
+    client_kwargs={'endpoint_url': 'https://renc.osn.xsede.org'}
+)
+```
+
+The enpoint URL will be given to you when your key ID and access key are assigned.
+
+## Writing Data to S3
+
+With greater permissions, you may be able to do more destructive activities (overwriting, removing, etc).
```

Comment:
greater permissions than what? What are default permissions for a new user? How would someone get these elevated permissions?

Reply:
greater permission than read-only/anonymous.